### ISSUE TITLE
добавить счетчик подписчиков suscribe

### ISSUE THREAD

**Issue by EugeneKormin** [EugeneKormin/Game-exe#158]
<a target="_new" href="http://subscribe.ru/catalog/comp.news.panda"><img
alt="Рассылка 'Новости и рассуждения от Панды'"
border="0" width="88" height="31"
src="http://subscribe.ru/stat/comp.news.panda/count.png?rating.lite"></a>

**Comment by simlex-titul2005** [comment]
На какое место поставить?

**Comment by EugeneKormin** [comment]
Рядом с соц кнопками.

**Comment by simlex-titul2005** [comment]
![default](https://cloud.githubusercontent.com/assets/8962193/14877945/ff115bee-0d28-11e6-9da2-71a3ad578a5a.png)

здесь?

**Comment by EugeneKormin** [comment]
Нет, с кнопками, которые сами делаем, помнишь, решали, что любая шара с соц сети, кроме нашей - это расшаривание, Яндекс ещё Твиттер считать не может, а мы смогли.

**Comment by EugeneKormin** [comment]
Вот рядом с кнопками, которые считают расшаривание статьи

**Comment by EugeneKormin** [comment]
Рядом с кнопками, которые предлагают присоединиться к группе в соцсети.

**Comment by EugeneKormin** [comment]
К нашим группам
